autogluon/task/tabular_prediction/predictor.py

```diff
@@ -129,15 +136,21 @@ def predict_proba(self, dataset, model=None, as_pandas=False):
                 Valid models are listed in this `predictor` by calling `predictor.model_names`
             as_pandas : bool (optional)
                 Whether to return the output as a pandas object (True) or numpy array (False).
-                Pandas object is a DataFrame if this is a multiclass problem, otherwise it is a Series.
+                Pandas object is a DataFrame if this is a multiclass problem or `as_multiclass=True`, otherwise it is a Series.
+                If the output is a DataFrame, the column order will be equivalent to `predictor.class_labels`.
+            as_multiclass : bool (optional)
+                Whether to return binary classification probabilities as if they were for multiclass classification.
+                    Output will contain two columns, and if `as_pandas=True`, the column names will correspond to the binary class labels.
+                    The columns will be the same order as `predictor.class_labels`.
+                Only impacts output for binary classification problems.
 
             Returns
             -------
             Array of predicted class-probabilities, corresponding to each row in the given dataset.
-            May be a numpy Ndarray or pandas Series/Dataframe depending on `as_pandas` argument and the type of prediction problem.
+            May be a numpy ndarray or pandas Series/DataFrame depending on `as_pandas` and `as_multiclass` arguments and the type of prediction problem.
```

Comment:
Add clarification: "For binary classification problems, this will return pandas Series containing for each datapoint only the predicted probability of the positive class, unless you specify `as_multiclass = True`."

Reply:
Added